### DISCUSSION TITLE
AG monitor Red Dot

### DISCUSSION THREAD

**Discussion by aluntay** [apohl79/audiogridder#867] [Help]
![Screenshot 2021-12-28 at 11 12 39](https://user-images.githubusercontent.com/96002239/147560605-cb8f8ae7-cf77-4caf-804e-b3c765935303.png)

Hi, I've just reopened a session on my Mac and some of the AG instances (the server is on a separate PC) either cannot open at all or seem to work but the plugin wont open. Any ideas please? Thanks

What is the red dots representing in under "Perf" on the AG Monitor. Also, to reiterate, Crimson for example looks fine in the Monitor but I can't actually open the plugin itself?

ps I'm using the latest beta version and the red dot in the "Perf" part flashes red and green, not fixed on red. What is this suggesting?

**Comment by Henri75** [comment]
Hi,

Red dots means connection issue. Blinking means it connects, then disconnects.

From my experience there is two possible reasons : 

1) AG server crashes and reloading (happens more often on mac client or server, windows seems more stable)

2) Missing plugin. Beware with beta6 some plugins ID changed.
With only 1 plugin having problem on a chain, the whole chain wont connect (Andreas is aware  about that and should fixed it soon hopefully)

Check the log on the server side, you will find info about what is causing the issue (for example unable to load a plugin).

**Comment by fojerhar** [comment]
HI
I have ran into the same problem. Updated my system from Catalina to Monterey, new projects are working fine, but cannot open the old ones, extreme long delays show up (1150ms) and red dots and protools mostly freezes also.
Would be nice to have some solution for this.
Thank you
Great product!

**Reply by fojerhar** [reply]
Here are the log files for this
[AudioGridder.zip](https://github.com/apohl79/audiogridder/files/8221801/AudioGridder.zip)
I have tried reversed, made a session in Monterey, and it runs well in catalina os

**Reply by fojerhar** [reply]
Made some investigation.
Some of the plugins were badly installed, reinstalling helped in a few things.
Now if I have a session what has more chains (sandboxes) protools freezes with beachball. (it happens when the configuration file changed while PT is open.
If I open the session, making inactive all the AG plugins, then I start the server and activate the chains one by one it can be done.

Having this lines mainly:
loading Pro-Q 3 (VST3-Pro-Q 3-8eacb89) [on connect]... 
[22:23:17.971|client|7fce142d81d0] failed to read parameters: EC=E_DATA STR=failed to read message header
[22:23:17.971|client|7fce142d81d0] screen receiver failed to read message: EC=E_DATA STR=failed to read message header
[22:23:17.971|client|7fce142d81d0] ...failed: failed to read parameters: EC=E_DATA STR=failed to read message header

error: instance (Gullfoss Master > Virtual Tape Machines > OPALEQ > soothe2 > ERINEQST > EMERALD2 > INDIGOEQ > TAUPETAPEZL > Oxford Inflator > Ozone 9): waitRead failed
[22:32:29.359|client|7fbc49ab29d0] screen receiver failed to read message: EC=E_DATA STR=failed to read message header
[22:32:29.359|client|7fbc49ab29d0] screen receiver terminated
[22:32:30.150|client|7fbc49ab29d0] failed to get result: EC=E_DATA STR=failed to read message header
[22:32:30.150|client|7fbc49ab29d0] ...failed: failed to get result: EC=E_DATA STR=failed to read message header
[22:32:30.150|client|7fbc49ab29d0] loading ERINEQST (VST3-ERINEQST-bbdc95b5) [on connect]... 

What does this mean?
(AMD hackintosh, 12.1 monterey, PT 2021.12

**Reply by apohl79** [reply]
The server failed to load ERINEQST, or at least it took too long, so the client (AG plugin) run into a timeout. You can check the server log, it might reveal more details but it might just "hang" at the load step. If you see that the load finished eventually followed by a client read error (because the client hang up after the timeout), you could try to increase the timeout value of the AG plugin. For this, open the config file (https://audiogridder.com/plugin/#config) and increase the value of "LoadPluginTimeoutMS". It is 15 seconds by default.

**Reply by fojerhar** [reply]
Hi Andreas, thank you, tried it (make the timeout longer, but could not help)
I reinstalled the monterey totally, and started from scratch. First moments were looking great, but as I installed all the (200Gb) plugins, issues started to come.As i open chains into a new session, all is fine. As I close it and open back start to have red dots (sometimes not 1 of 10 attempts) - extreme long timings (like 5-6ms is standard, and here I had 1120ms..., and while server starts to clean up things it is too long waiting time for protools, and PT freezes with beachball. After 30-60secs I can see the timings are well at plugin tray, but this is too late for pt.
[AudioGridderPlugin_2022-03-16_12-03-32(2) protools.log](https://github.com/apohl79/audiogridder/files/8262171/AudioGridderPlugin_2022-03-16_12-03-32.2.protools.log)
Tried to make fake sessions with reaper. The same happens, but reaper doesnt freeze, and after a minute all the plugins runs well
[AudioGridderPlugin_2022-03-16_11-36-16 reaper.log](https://github.com/apohl79/audiogridder/files/8262175/AudioGridderPlugin_2022-03-16_11-36-16.reaper.log)
In the log can be seen the attempts.
Meanwhile at Catalina AG works like a charm, loads fast, so this could be a monterey issue.

Strange, lots of plugins runs well in AG what doesnt have the monterey updated version.... :)


So which plugins fail in protools, the same is working in reaper after 60 secs.

thank you

**Comment by Henri75** [comment]
I think it is related to some issues I already reported.

I encountered too some reconnection issues when I open a project on mac
with mac servers.
I dont have anymore red dots, but something new with latest AG : green dot
but in fact the connection is not ok, like an in between state.

Ìt seems to happen with a lot of  AG inserts.

Never seen it with only a few AG inserts.

If I open all AG insert and manually reconnect then everything is fine.

My guess is that it may be due to some kind of temporary overload/timeout
when instantiating/opening all those connections, or some heavy
plugins chain on the servers.



Le dim. 13 mars 2022 à 22:34, Viktor Szabó ***@***.***> a
écrit :
